src/Template/Element/advanced_search_form.ctp

```diff
@@ -357,6 +357,11 @@ echo $this->Form->create('AdvancedSearch', [
                    href="http://en.wiki.tatoeba.org/articles/show/text-search">
             <?= __('More search options') ?>
         </md-button>
+        <?php if ($layout == 'row'): ?>
```

Comment:
Your code would be easier to understand if you used `$isSidebar` instead of `$layout == 'row'`.